libs/blocks/accordion/accordion.js

```diff
@@ -10,7 +11,40 @@ function setSEO(questions) {
   document.head.append(script);
 }
 
-function handleClick(el, dd) {
+function displayMedia(displayArea, el, dd, i, id, expanded) {
+  const theID = el.getAttribute('aria-controls').split('-')[1];
+  [...mediaCollection[theID]].map(
+    (mediaCollectionItem, idx, total) => {
+      mediaCollectionItem.classList.remove('expanded');
+
+      for (let index = 0; index < total.length; index++) {
+        mediaCollectionItem.classList.remove('expanded');
+        const trigger = document.querySelector(`#accordion-${theID}-trigger-${index + 1}`);
+        const content = document.querySelector(`#accordion-${theID}-content-${index + 1}`);
+        trigger.setAttribute('hidden', '');
```

Comment:
we can convert L#24 - 25 to a method and then apply to `trigger` and `content`

Reply:
Agreed - that could be used for all of the toggling on lines below too